meta-mel/conf/distro/mel.conf

```diff
@@ -187,6 +187,14 @@ require conf/distro/include/yocto-uninative.inc
 
 INHERIT += "uninative"
 ## }}}1
+## Mechanisms provided for user customization {{{1
+# Support USER_FEATURES
+INHERIT += "user_features"
```

Comment:
What's the relevance with user features here?

Reply:
Just moved from a different part of the file to consolidate user customization hooks/mechanisms.